de4dot.code/deobfuscators/Dotfuscator/CflowDecrypter.cs

```diff
@@ -16,7 +16,7 @@
     You should have received a copy of the GNU General Public License
     along with de4dot.  If not, see <http://www.gnu.org/licenses/>.
 */
-
+using System.Collections.Generic;
```

Comment:
This using statement isn't needed. Remove it and keep the empty line you removed.

You just add a new commit and push it. You don't have to create a new PR, github will add your changes in this PR.

Reply:
ok,thanks.I will keep the empty line,but the using statement is needed by the method GetFixIndexs,and how to modify the codes?help me please.

Reply:
My bad, I thought you only added it for List<T> but in that case keep it but restore the empty line. :)

Reply:
ok，iwill do it。